src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactions.tsx

```diff
@@ -27,6 +26,25 @@ const Container = styled(Box)(() => ({
   overflowY: 'auto',
 }));
 
+/**
+ * Converts the "amount" string to a number to remove ".0"
+ * If the value is 0 or isExpense is true, it returns the value as is.
+ * Otherwise, it removes the '-' character if present, or prepends it if absent.
+ */
+const formatAmountForExport = (
+  amount?: string | null,
+  isExpense?: boolean,
+): number => {
+  if (!amount) {
+    return 0;
+  }
+
+  if (amount === '0' || isExpense) {
+    return Number(amount);
+  }
+  return amount?.[0] === '-' ? Number(amount.substring(1)) : -Number(amount);
+};
+
 const formatDateRange = (startDate?: DateTime, endDate?: DateTime) => {
   if (!startDate) {
```

Comment:
I just noticed that we're reassigning the arguments here. Can you make a different variable? Or give them a default value since that's the intent of this code (this might be a bad idea since the start date expression is a little complex).

```ts
const formatDateRange = (
  startDate: DateTime = DateTime.local().minus({ months: 1 }).plus({ days: 1 }),
  endDate: DateTime = DateTime.local(),
) => `${startDate.toISODate()}..${endDate.toISODate()}`;
```